biothings/hub/databuild/builder.py

```diff
@@ -864,6 +864,8 @@ async def merge_source(self, src_name, batch_size=100000, ids=None, job_manager=
         meta = src_master.find_one({"_id": src_name}) or {}
         merger = meta.get("merger", "upsert")
         self.logger.info("Documents from source '%s' will be merged using %s", src_name, merger)
+        merger_keywords = meta.get("merger_keywords")
+        self.logger.info("Documents from source '%s' will be using these extra parameters during the merge %s", src_name, merger_keywords)
```

Comment:
add `if merger_kwargs:` (after renaming) here, or you might delete the whole log line if it meant to be temp.